.github/workflows/ci.yml

```diff
@@ -0,0 +1,74 @@
+name: CI
+
+on:
+    push:
+        branches: [ master ]
+    pull_request:
+    schedule:
+        -   cron: '0 0 * * MON'
+
+jobs:
+    check-cs:
+        name: Check Coding Standards
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+
+            - name: PHP-CS-Fixer
+              uses: docker://oskarstark/php-cs-fixer-ga
+              with:
+                args: --config=.php-cs-fixer.php --diff --dry-run
+
+    ci:
+        name: Test PHP ${{ matrix.php-version }} ${{ matrix.name }}
+        runs-on: ubuntu-latest
+        strategy:
+            fail-fast: false
+            matrix:
+                php-version: ['7.4', '8.0']
+                composer-flags: ['']
+                name: ['']
+                include:
+                    - php-version: 7.4
+                      composer-flags: '--prefer-lowest'
+                      name: '(prefer lowest dependencies)'
+        services:
+          elasticsearch:
+            image: docker.elastic.co/elasticsearch/elasticsearch-oss:7.8.0
+            ports:
+              - '9200:9200'
+        steps:
+            - name: Wait for ES to be ready
+              run: |
+                curl 127.0.0.1:9200 || true
+                curl localhost:9200 || true
+                docker ps
+                wget -q --waitretry=1 --retry-connrefused -T 10 -O - http://127.0.0.1:9200
```

Comment:
We need a green cluster, not only a responding one.

Maybe you could use what was used in travis?

https://github.com/jolicode/elastically/blob/446d90fda74bc86ece9dcec50733f2131a204682/.travis.yml#L23-L26

Reply:
✅